pkg/types/interfacetests/chain_reader_interface_tests.go

```diff
@@ -16,6 +16,8 @@ import (
 
 type ChainReaderInterfaceTester[T TestingT[T]] interface {
 	BasicTester[T]
+	Start(t T)
+	Close(t T)
```

Comment:
Why do we need this?

Reply:
Thanks for pointing this out. We just need `Close()`. 

The idea is to be able to `Close` the service before calling its methods and check for the error